packages/alea-frontend/pages/api/get-fau-clip-info/[clipId].ts

```diff
@@ -19,21 +19,31 @@ function isFresh(cachedObject: CachedObject) {
 export async function getVideoInfo(clipId: string): Promise<ClipDetails> {
   const url = `${FAU_TV_OEMBED_BASE_URL}?url=${FAU_TV_BASE_URL}/clip/id/${clipId}&format=json`;
   const { data } = await axios.get(url);
-//TODO: Hack: for now FAU TV api provides 360p,720p and 1080p url for only presenter view
-//TODO:  In both presenter+presentation view FAU TV provide only one url , so adding that only as of now.
+  //TODO: Hack: for now FAU TV api provides 360p,720p and 1080p url for only presenter view
+  //TODO:  In both presenter+presentation view FAU TV provide only one url , so adding that only as of now.
+  const presenterUrl: string | undefined =
+    (data && (data.presenter_url as string)) || (data && (data.file as string)) || undefined;
+  const presentationUrl: string | undefined =
+    data && (data.presentation_url as string) ? (data.presentation_url as string) : undefined;
+  const compositeUrl: string | undefined =
+    data && (data.composite_url as string) ? (data.composite_url as string) : undefined;
+
   const videoInfo: ClipDetails = {
     // r360: data.alternative_Video_size_small_url || undefined,
     // r720: data.alternative_Video_size_medium_url || undefined,
     // r1080: data.alternative_Video_size_large_url || undefined,
-    r360: data.file|| undefined,
-    r720: data.file || undefined,
-    r1080: data.file || undefined,
+    r360: presenterUrl,
```

Comment:
So, quality is effectively useless? We should remove the data and the feature in that case